runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -1000,6 +1000,20 @@ public void setZOrderOnTop(boolean onTop) {
         mContent.setZOrderOnTop(onTop);
     }
 
+    /**
+     * Removes the autocomplete popup from the currently focused form field, if present.
+     * Note this only affects the display of the autocomplete popup, it does not remove
+     * any saved form data from this WebView's store.
+     * This is a poorly named method, but we keep it for historical reasons.
+     * @since 6.0
+     */
+    @XWalkAPI
+    public void clearFormData() {
+        if (mContent == null) return;
+        checkThreadSafety();
+        mContent.hideAutofillPopup();
```

Comment:
For most case, this should be OK.  But if  clearFormData is called before load, this will not work.
As a workaround , you may refer to XWalkContent.setBackgroundColor